HelloDotNet/HelloDotNet.csproj

```diff
@@ -5,7 +5,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LaunchDarkly.ServerSdk" Version="5.*" />
-    <PackageReference Include="Common.Logging" Version="3.*" />
+    <PackageReference Include="LaunchDarkly.ServerSdk" Version="6.0.0-rc.1" />

```

Comment:
This exact-version dependency will be changed to `6.*` after the GA release, so the demo will always pick up the latest GA version.